process/block/preprocess/transactions.go

```diff
@@ -907,7 +907,8 @@ func (txs *transactions) createAndProcessMiniBlocksFromMe(
 			}
 
 			numTxsBad++
-			log.Trace("bad tx",
+			//TODO: Change log level back to Trace
```

Comment:
too many todo's. after testing this branch may be deleted.

Reply:
This branch should be merged in dev for now, as there are other PRs in waiting list to be merged in dev, and all the tests are done using dev so we need these prints for now in dev to catch the problem if it will be reproduced again